hooks/cascading-scans/Makefile

```diff
@@ -54,15 +56,5 @@ deploy:
 	helm -n integration-tests upgrade --install dssh . --wait \
 		--set="hook.image.repository=docker.io/$(IMG_NS)/$(hook-prefix)-$(name)" \
 		--set="hook.image.tag=$(IMG_TAG)"
-
-integration-tests:
-	@echo ".: 🩺 Starting integration test in kind namespace 'cascading-scans'."
-	kubectl -n integration-tests delete scans --all
-	cd ../../tests/integration/ && npm ci &&	npx --yes --package jest@$(JEST_VERSION) jest --verbose --ci --colors --coverage --passWithNoTests scanner/cascade-nmap-ncrack.test.js
-
+		
 deploy-test-deps-2: deploy-test-dep-unsafe-https deploy-test-dep-nmap deploy-test-dep-sslyze
```

Comment:
In general make uses the target name as file name, e.g. you can have a target `*.ts` which will be invoked for all TypeScript files. You use this to call the compiler in the target. Typical targets from C are `*.c`, *.o` etc. All targets which are not file names are so called [phony targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) and must be tagged with `.PHONY`. Unless we do this a target may be invoked ifsoeone adds a file which accidentally match the target name.

Please add `.PHONY` to all targets.